amr-wind/incflo_compute_dt.cpp

```diff
@@ -112,6 +112,11 @@ void incflo::ComputeDt(bool explicit_diffusion)
                         result = std::abs(v_bx(i, j, k, 0)) * dxinv[0] / fac_x +
                                  std::abs(v_bx(i, j, k, 1)) * dxinv[1] / fac_y +
                                  std::abs(v_bx(i, j, k, 2)) * dxinv[2] / fac_z;
+
+                        // Multiply advective CFL by 2 when near interface
+                        result *= 2.0;
```

Comment:
Hmm is this 0.5, to do with the mass-momentum consistency criterion?

Reply:
pretty much -- the VOF scheme needs CFL < 0.5 to ensure conservation of mass. By multiplying this (the calculated advective CFL) by 2, that criterion becomes CFL < 1.0.

Essentially, with this change, the user can enforce CFL < 0.5 near the interface and enforce CFL < 1 elsewhere by setting the max CFL to 1 in the input file.

Reply:
okay, make sure you remove any restrictions for CFL=0.5 from other parts of the code.

Reply:
That's what I did with the reg tests. This change doesn't apply to the CFL calculation when the velocities are prescribed, so I didn't change the reg tests with prescribed velocities.